### PR TITLE
libpod: Add definition of containerPlatformState for FreeBSD

### PR DESCRIPTION
**Pull request by dfr**
For FreeBSD, we need the name of the 'network jail' which is the parent
of all containers in a pod. Having a separate jail for the network
configuration also simplifies the implementation of CNI plugins so we
use this pattern for solitary containers as well as pods.

[NO NEW TESTS NEEDED]

Signed-off-by: Doug Rabson <dfr@rabson.org>

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by mheon** [comment]
/approve
LGTM
@containers/podman-maintainers PTAL

**Comment by dfr** [comment]
Fix a mis-wording in the commit message

**Review by vrothberg — APPROVED** [review]
/lgtm
/hold

**Comment by umohnani8** [comment]
LGTM

**Pull request merged**
